VOKMockUrlProtocol.m

```diff
@@ -49,6 +49,12 @@ + (instancetype)containerWithResponse:(NSHTTPURLResponse *)response data:(NSData
 
 @implementation VOKMockUrlProtocol
 
+static NSBundle *testBundle = nil;
```

Comment:
Because if for some reason someone nulls it out at any point I want it to pick up the test bundle again - although I suppose I could handle that case in `setTestBundle` too...